### DISCUSSION TITLE
Blue Link Labs funding & organizational structure

### DISCUSSION THREAD

**Discussion by willurmston** [beakerbrowser/beaker#1751] [General] [answered]
Not sure if this is the right place for this but here goes. How are you thinking about the long-term organization of the Beaker project? The thought of a handful of SV investors having outsized control over Beaker 10 years from now makes me sad, given the democratic philosophy at its core. How are you planning to keep Beaker true to its values and ensure its long-term stability?

From what I can tell, it seems like you aren't profit-oriented, which is great. Aside from being a not-for-profit like Signal Foundation, have you considered forming a worker cooperative? The democratic structure would be a good fit for something community-driven like Beaker, and you may be able to get some funding just for being a co-op. In my view, software co-ops would help solve a lot of problems in the tech industry, from pay discrimination, to growth-obsession, to being pressured to make drone targeting software/oil-extraction algorithms/police surveillance tools. It seems fitting for Blue Link Labs to try experimenting with new ways of organizing tech work.

If you don't know about them, here is [an entertaining and informative lecture](https://www.youtube.com/watch?v=ynbgMKclWWc) about worker cooperatives. 

I'm sure you're considering many options, just thought I would mention it 🙂

**Comment by pfrazee** [comment, marked as answer]
We're basically open to any model that's going to give Beaker the resources it needs to succeed. We've been extremely shoe-string so far, and the team has done great work under that constraint, but it's a challenge and I think it will inhibit our growth if we don't have good capital sources soon.

Currently we're a Deleware C-Corp and have raised ~200k in angel funding. We think we have an ethical and effective business model planned in the form of selling compute and storage -- at first via the cloud as co-hosting services (think "AWS for users") and then eventually as hardware you can run at home. The idea being, users will need more powerful *personal* computing resources as we move more of the stack into their hands, and so we can provide that. We're focused very heavily on anti-lockin (it's frictionless to switch co-hosting services) because that's a pretty core part of the open computing ethos.

Connected to that, we open-source everything you need in this stack to run it independently or fork entirely. We do that for a lot of reasons, but one of them is to protect everyone against our organization -- particularly once we're no longer involved.

SV has earned its bad reputation with VC and that behavior is arguably a part of why this project was created. Personally I think the foundational issue is business model; Facebook and Google steal data because that's how they make money. While I don't want to hold up Apple as a set of saints, they've been able to respond _more_ (not enough but more) to users' privacy because they make their money with hardware, not information. Selling computing resources is one of the few high-margin businesses in tech, and that's why I believe it can fund this work ethically.

I have some experience with both non-profit and for-profit, and both are challenging, but it was angel investors who understood what we were doing enough to fund us in the early stages. I have a similar kind of pragmatism with organizational models going forward: I care about what works for this project. I feel like we put our protections in other places -- business model, code licensing, and open architectures -- so when it comes to funding and organization, we have space to be as pragmatic as possible.

I'm watching that talk and I'm open to any proposals that have real legs, but it's gotta really have signs that we can get capital soon and at the amounts that a project this ambitious requires.

**Reply by willurmston** [reply]
Thanks for your thorough response, it's clear you've thought it through. I'm happy you are putting so many self-checks in place and if you can convince people to invest without compromising on them, good on you. I wish there was a clear path outside of VC. Thank you for considering a co-op model at the very least. If you could swing it it would be a game changer for the product, employees, and the industry.

P.S. The idea of open source Beaker hardware down the road is very exciting. I would absolutely buy a little co-hosting server.

**Reply by pfrazee** [reply]
Cheers! I should have mentioned, the Hypercore Protocol tech should move to a non-profit or similar when we have the resources to form it. I hope I'm right too! Worst case, somebody can fork us.
